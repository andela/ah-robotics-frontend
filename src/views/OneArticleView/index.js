import React, { Component } from 'react';
import RatingsView from '../RatingsView';

class OneArticleView extends Component {
    state = { }

    render() {
        return (
          <div>
            <RatingsView />
          </div>
        );
    }
}

export default OneArticleView;
