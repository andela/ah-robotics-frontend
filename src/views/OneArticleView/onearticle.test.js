import React from 'react';
import OneArticleView from '.';

it('app should match the snapshot', () => {
    const wrapper = shallow(<OneArticleView />);
    expect(wrapper).toMatchSnapshot();
  });
