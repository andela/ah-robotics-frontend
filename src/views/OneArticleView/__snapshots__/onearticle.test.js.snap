// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match the snapshot 1`] = `
<div>
  <Connect(RatingsView) />
</div>
`;
